package.json

```diff
@@ -110,6 +110,7 @@
     "test:browser": {
       "command": "karma start",
       "dependencies": [
+        "clean-test-failures-file",
```

Comment:
retries only work with the `test:browser-ci:retry` script so we delete this file on non-CI runs.